src/Sentry/Protocol/Device.cs

```diff
@@ -209,10 +209,12 @@ public sealed class Device : IJsonSerializable
         /// </summary>
         /// <example>
         /// iOS: UIDevice.identifierForVendor (UUID)
-        /// Android: md5 of ANDROID_ID
+        /// Android: The generated Installation ID
         /// Windows Store Apps: AdvertisingManager::AdvertisingId (possible fallback to HardwareIdentification::GetPackageSpecificToken().Id)
         /// Windows Standalone: hash from the concatenation of strings taken from Computer System Hardware Classes
         /// </example>
+        /// TODO: Investigate - Do ALL platforms now return a generated installation ID?
```

Comment:
They should. ANDROID_ID isn't privacy aware, different apps can get the same value and 'link' a user across apps.

Our goal is not to use such ids. Apple now asks for permission for accessing the "advertising id" which we don't use in the iOS SDK.